src/www/services_rfc2136.php

```diff
@@ -140,7 +140,7 @@
                       <td>
 <?php
                         $filename = "/conf/dyndns_{$rfc2136['interface']}_rfc2136_" . escapeshellarg($rfc2136['host']) . "_{$rfc2136['server']}.cache";
-                        if (file_exists($filename) && !empty($rfc2136['enable'])) {
+                        if (file_exists($filename) && !empty($rfc2136['enable']) && $rfc2136['recordtype'] != "AAAA") {
```

Comment:
same logic as in services.inc should be used here. We cannot expect it to be set, it would probably produce a warning now.